README.md

```diff
@@ -43,6 +43,12 @@ Collection of awesome Python stubs and tools to work with them.
 - [pytype](https://github.com/google/pytype/) - A static type analyzer, alternative to mypy. Can do static type inference, and generate stub files with infered types.
 - [merge_pyi](https://github.com/google/pytype/tree/master/pytype/tools/merge_pyi) - part of pytype toolchain, applies stub files onto source code.
 
+## Integrations
```

Comment:
Please, also list this section and `Mypy plugins` in `Contents`

Reply:
:heavy_check_mark: